.github/workflows/task-get-linux-configurations.yml

```diff
@@ -37,6 +37,7 @@ on:
 
 jobs:
   get-required-envs:
+    name: for ${{ inputs.platform }} platform on ${{ inputs.architecture }} architecture
```

Comment:
Consider prepending `Get Linux Configurations`.

Reply:
Because this workflow is triggered by `workflow_call` only, the full name will be clear from the context